Sources/ArcGISToolkit/Components/Augmented Reality/WorldScaleSceneView.swift

```diff
@@ -17,6 +17,8 @@ import SwiftUI
 import ArcGIS
 
 /// A scene view that provides an augmented reality world scale experience.
+@MainActor
+@preconcurrency
```

Comment:
The `MainActor` attribute adds a new constraint to the type. Since the type is public, it may be used in non-main actor contexts or a user may have extended it with code that may now have build warnings or errors due to the new constraint. So we add the `preconcurrency` attribute to public methods to prevent the new constraint from causing problems unless and until the code that uses it is built with strict concurrency checking. For more information, see [the documentation](https://docs.swift.org/swift-book/documentation/the-swift-programming-language/attributes/#preconcurrency) for the attribute.